extensions/opentelemetry/opentelemetry-exporter-otlp/runtime/src/main/java/io/quarkus/opentelemetry/exporter/otlp/runtime/OtlpExporterUtil.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.opentelemetry.exporter.otlp.runtime;
+
+import java.util.AbstractMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import io.quarkus.opentelemetry.runtime.tracing.TracerUtil;
+
+public class OtlpExporterUtil {
+
+    private OtlpExporterUtil() {
+
+    }
+
+    /**
```

Comment:
Can we extract the core functionality from this method, which was in turn taken from `TracerUtil`, to a method in maybe `OpenTelemetryUtil` in the OpenTelemetry extension? Possibly called `convertKeyValueListToMap`, or a better name.

Then modify `TracerUtil` to call the new util method, and adding the attribute handling bit around it. Then this method calls the new method on `OpenTelemetryUtil`?

The comment on this method needs updating as well

Reply:
Please see the new method `OpenTelemetryUtil. convertKeyValueListToMap ()`